tests/testthat/test-fill.R

```diff
@@ -15,25 +15,23 @@ test_that("all missings left unchanged", {
   expect_identical(up, df)
 })
 
-test_that("missings filled down from last non-missing", {
+test_that("missings are filled correctly", {
+  # filled down from last non-missing
   df <- tibble(x = c(1, NA, NA))
   out <- fill(df, x)
   expect_equal(out$x, c(1, 1, 1))
-})
 
-test_that("missings filled up from last non-missing", {
+  # filled up from last non-missing
   df <- tibble(x = c(NA, NA, 1))
   out <- fill(df, x, .direction = "up")
   expect_equal(out$x, c(1, 1, 1))
-})
 
-test_that("missings filled down-and-then-up from last non-missing", {
+  # filled down-and-then-up from last non-missing
   df <- tibble(x = c(NA, 1, NA, 2, NA, NA))
```

Comment:
Could you please tweak all the tests to use a single data frame? (i.e. this one).